src/components/Result.jsx

```diff
@@ -177,17 +170,17 @@ const Events = ({data}) => {
 	}
 };
 
-const Weather = ({data}) => {
-	data = data[0];
```

Comment:
Does both usecases work without this? (single day, range of days)

Reply:
Yes I checked before pushing. It should definitely work